modules/client-common/src/main/java/org/apache/ignite/internal/client/proto/ClientBinaryTupleUtils.java

```diff
@@ -49,72 +79,251 @@ public static void readAndSetColumnValue(
         }
 
         switch (clientDataType) {
-            case ClientDataType.INT8:
+            case INT8:
```

Comment:
Is importing all the enum values separately required by the coding convention?

Reply:
Coding convention does not mandate this, as far as I understand.